articles/multifactor-authentication/overview.md

```diff
@@ -0,0 +1,103 @@
+
+* **Knowledge**: Something the user **knows** (such as a password)
+* **Possession**: Something the user **has** (such as a mobile device)
+* **Inheritance**: Something the user **is** (such as a fingerprint or retina scan)
+
+## Implement MFA with Auth0
+
+Enabling MFA for your tenant is a fairly straightforward process. First, you toggle on the factors you choose to enable on your tenant, such as push notifications or SMS. Next, you perform any further setup required for that factor, and last, you choose whether you wish to force MFA for all users or not. See the instructions below for details.
+
+You can also [customize your MFA flow](/multifactor-authentication/custom) with [Auth0 Rules](/rules), to allow MFA to only be required in specific circumstances or force a particular factor to be used.
+
+### 1. Toggle on the factors you require in the Dashboard
+
+In the [Dashboard > Multifactor Auth](${manage_url}/#/mfa), head to the Multifactor Auth section. Here you will find a series of toggles for the MFA factors supported by Auth0. 
+
+**TODO: STANDARDIZE DASHBOARD IMAGE**
+
+![MFA Dashboard Page](/media/articles/multifactor-authentication/mfa-dashboard.png)
+
+Any or all of these factors can be enabled simultaneously. When using Universal Login, the user will be shown a default and but will be allowed to choose another factor to use, if you have more than one factor enabled in the Dashboard.
+
+**TODO: IMAGE OF CURRENT LOGIN PROMPT WITH MFA CHOICES**
+
+::: note
+When you enable the Duo factor, you will have to fill in a few further settings related specifically to Duo before continuing.
+**TODO: SPECIFY SETTINGS**
+:::
+
+Additionally, there is the **Always require Multi-factor Authentication** toggle, which allows authentication against this app only when one or more of the factors is enabled in the Dashboard, and completed by users while logging in.
+
+### 2. Set up your services
+
+Auth0 supports the following factors for implementing MFA. You must enable at least one to use MFA, but you can choose to enable and make available more than one factor if you wish.
+
+#### One Time Passwords (OTP)
+
+The principle behind OTP as a factor is fairly straightforward for the end user. They use an app, such as Google Authenticator ([Google Play](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) / [App Store](https://itunes.apple.com/us/app/google-authenticator/id388497605?mt=8)). Upon signup, they can scan a code and set up the app, upon which it will begin generating one-time codes. Afterword, the user can simply check the app for the current one-time code to enter when authenticating using this factor.
+
```

Comment:
Can we also suggest Guardian, Authy, Microsoft Authenticator?

Reply:
Also added a note on our test plan to make sure the code can be scanned with the four apps.

Reply:
Let me know and I can reword this to be more specific about each provider if needed. While I'm getting screenshots I can check on a couple possibly.

Reply:
Afterword > Afterwards